### DISCUSSION TITLE
Routing for pages

### DISCUSSION THREAD

**Discussion by MrPrecise** [AutoPacker-OSS/AutoPacker#85] [General]
Hey,

I was making a NewOrganization.js, similar to NewServer.js and NewProject.js to help create a form to create new organizations.
Then I wanted to connect the file so when I click on a button "New Organization" in "Your Organizations", I will get directed to this page.  

But I had some problems figuring out what way this project were routing between pages, I was looking up some tutorials but they were often very different than what I could find in this project. 
Could anyone bring me in the right direction?

**Comment by ANicholasson** [comment]
For routing, we are using the "famous" react-router (very original). The routes are configured inside the App.js file/component. You should take a look at what has been done with the project and server scenarios which are quite similar. If you really want to dive into the routes as well, you can see that there are different types of routes: Route, ProfileDashboardRoute, OrganizationDashboardRoute. This should be self-explanatory, but they are the wrapper routes that are used for their respective scenarios. Might especially want to take a good look at the OrganizationDashboardRoute as you will most likely have to change this when implementing the "new" organization UI.
